gslab_scons/tests/test_size_warning.py

```diff
@@ -260,38 +326,40 @@ def isfile_ignored_side_effect(*args, **kwargs):
 
 
 #== Side effects for testing create_size_dictionary() ===
-# These functions mock a directory containing files of various sizes
-# and a system that recognises no directory other than that one.
+# These functions mock two directories containing files of various sizes
+# and a system that does not recognises any other directories.
 def isdir_dict_side_effect(*args, **kwargs):
     '''
-    Mock os.path.isdir() so that it only recognises ./test_files/
-    as an existing directory.
+    Mock os.path.isdir() so that it only recognises a few mocked directories
+    as existing directories.
     '''
     path = args[0]
     if not isinstance(path, str):
         raise TypeError('coercing to Unicode: need string or buffer, '
                         '%s found' % type(path))
-    return path in ['test_files', 'release']
+    acceptable = ['test_files', 'test_files/size_test', 'release', '.']
+    return os.path.normpath(path) in acceptable
 
 
 def walk_dict_side_effect(*args, **kwargs):
```

Comment:
I combined them. I think the mocking in this script could be further polished, although I'm unsure whether it's worth spending time to make improvements at this point.